lookout/style/typos_checker/analyzer.py

```diff
@@ -20,33 +20,34 @@ class CommentsCheckingAnalyzer(Analyzer):
     @with_changed_uasts_and_contents
     def analyze(self, ptr_from: ReferencePointer, ptr_to: ReferencePointer,
                 data_request_stub: DataStub, **data) -> [Comment]:
+        commentFilter = "//*[@roleComment]"
         changes = data["changes"]
         comments = []
 
         for change in changes:
-            old_comments = set([node.token for node in bblfsh.filter(change.base.uast, "//*[@roleIComment]")])
-            comment_nodes = [node for node in bblfsh.filter(change.head.uast, "//*[@roleComment]")
+            old_comments = set([node.token for node in bblfsh.filter(change.base.uast, commentFilter)])
+            comment_nodes = [node for node in bblfsh.filter(change.head.uast, commentFilter)
                              if node.token not in old_comments]
             comment_tokens = [node.token for node in comment_nodes]
             if len(comment_tokens) > 0:
                 suggestions = self.model.correct_comments(comment_tokens)
                 for index, comment_node in enumerate(comment_nodes):
                     if index in suggestions.keys():
                         corrections = suggestions[index]
+                        line_suggestions = ""
                         for token in corrections.keys():
-                            comment = Comment()
-                            comment.file = change.head.path
                             corrections_line = ""
                             for candidate in corrections[token]:
-                                corrections_line += candidate[0] + " (%d)," % \
-                                                    int(candidate[1] * 100)
-                            comment.text = "Typo inside comment in token '%s'. " \
-                                           "Possible corrections:" % token +\
-                                           corrections_line[:-1]
-                            comment.line = comment_node.start_position.line
-                            comment.confidence = int(
-                                corrections[token][0][1] * 100)
-                            comments.append(comment)
+                                corrections_line += " `%s` (%d)," % \
+                                                    (candidate[0], int(candidate[1] * 100) )
+                            line_suggestions += "Typo inside comment in word `%s`. " \
```

Comment:
Maybe replace 'word' with 'token'? For me sounds more codeish, but it doesn't matter so much.

Reply:
Are we finding typos in (natural) words, or in (code) tokens?
If you still prefer using `token` just confirm me and I'll replace it :D

Reply:
Yes we do, it's just that i'm used to saying this to any text tokens even in natural text:) It really doesn't matter, leave it as it is if you think it sounds better.

Reply:
I'm ok with keeping the maintainer (your) point of view about this suggestion &mdash;and I have no strong opinion either&mdash; so unless @tsolakoua or @meyskens would prefer `word` rather than `token`, I'll replace it :dancer:

Reply:
somewhere i prefer `word`, the word `token` makes me think of oauth tokens or nonces since i used these a lot in a security related context